### DISCUSSION TITLE
Cart transform - Shopify Functions error notification in production

### DISCUSSION THREAD

**Discussion by AlexanderLindkjaer** [Shopify/function-examples#409] [Feedback: Cart Transform API]
I got 2 different apps containing cart transform functions, distributed to 2 different plus stores as a custom app. 

Both functions, while working in dev, does not return the expected operations in prod. 
As per the docs https://shopify.dev/docs/api/functions/errors i thrown an error manually to debug it. I now see the function failing in my partner dashboards for the plus stores. 

The docs says that there should be som kind of error notification in the shop admins where i can share the error repport with my partner dashboard. The docs says very vaguely "From your Shopify admin, navigate to the page associated with your executed function"

What page is associated with cart transforms (merge and expand operations)  and where exactly should this error notification pop up? i can't find the notification anywhere for the life of me 😄

**Comment by anthony-v-sundae-lane** [comment]
Just came here to log this issue / ask the question and found this post.

Any updates Shopify team?

**Comment by phpjscoding** [comment]
to find your Functions logs go to 'partners.shopify.com' -> apps -> all aps-> 'your app' -> extensions -> 'your function name' . there you find the list of logs list by date .
NOTE : only console.log(JSON.stringify(obj)) ... trying to log an obj may stop the function

**Reply by AlexanderLindkjaer** [reply]
@phpjscoding i think you misunderstood. This is not an issue of where to find logs in the partner dashboard. The issue is accessing logs on live stores.

**Comment by PattyKovash** [comment]
For anyone looking for an answer to this, this is currently not available for cart transform. From a thread in the Shopify Partners Slack Community, it is on the radar but no public roadmap or timeline shared at this time. This was from 2 months ago.

https://shopifypartners.slack.com/archives/C0225DTE1M1/p1705953581306209?thread_ts=1705937197.225839&cid=C0225DTE1M1

**Comment by dev-prakhar** [comment]
This is released now. Can't find anything in the documentation though. 
@nickwesselman can you point us to the documentation for this?

**Comment by kalenjordan** [comment]
I've seen this on one store that I manage but I'm not currently seeing it in another store where I know there are some recent cart transform errors.
